### DISCUSSION TITLE
Transition

### DISCUSSION THREAD

**Discussion by tetap** [vuejs/rfcs#721] [General]
I found that the Transition component uses requestAnimationFrame internally, and when this method is executed, if the current tab is not in focus, i.e. document.visibilityState ! = 'visible', the function will be paused or changed to 1s, and then executed again when the focus is regained, and then the transition animation will be flickering, which is very difficult. There is no better way. Make the transition animation work properly.

![image](https://github.com/user-attachments/assets/ff1ebe58-02d4-4ea5-992a-c12e2ff63fdf)

**Comment by posva** [comment]
Hi, thanks for your interest but Github issues are for bug reports and feature requests only. You can ask questions on the [forum](http://forum.vuejs.org/), the [Discord server](https://chat.vuejs.org/) or [StackOverflow](http://stackoverflow.com/tags/vue.js).
